gnucash/gnome/reconcile-view.c

```diff
@@ -501,25 +501,60 @@ gnc_reconcile_view_class_init (GNCReconcileViewClass *klass)
 }
 
 
-static void
+static gboolean
 gnc_reconcile_view_toggle_split (GNCReconcileView *view, Split *split)
 {
     Split *current;
 
-    g_return_if_fail (GNC_IS_RECONCILE_VIEW (view));
-    g_return_if_fail (view->reconciled != NULL);
+    g_return_val_if_fail (GNC_IS_RECONCILE_VIEW (view), FALSE);
+    g_return_val_if_fail (view->reconciled != NULL, FALSE);
 
     current = g_hash_table_lookup (view->reconciled, split);
 
     if (current == NULL)
+    {
         g_hash_table_insert (view->reconciled, split, split);
+        return TRUE;
+    }
     else
+    {
+        g_hash_table_remove (view->reconciled, split);
+        return FALSE;
+    }
+}
+
+
+/*****************************************************************************\
+ * gnc_reconcile_view_rec_or_unrec_split                                     *
+ *   insert or remove a child split from the list of splits to be reconciled *
+ *   (view->reconciled) so that all other splits in the same transaction     *
+ *   for the account being reconciled (including children), are the same     *
+ *   reconciliation state as the split that has  been toggled                *
+ *                                                                           *
+ * Args: view           - the view to use                                    *
+ *       split          - the split to be inserted or removed                *
+ *       reconcile      - TRUE=insert, FALSE=remove                          *
+ * Returns: void                                                             *
+\*****************************************************************************/
```

Comment:
Please format the comment with [Doxygen](http://www.doxygen.nl/) markup, particularly when you lead the comment with `/**` that flags it for Doxygen to process it:
* No need to repeat the function name
* No need to specify that it returns void (there's no need for that in a not-Doxygen comment either, it's obvious from the function signature).
* Begin each parameter line with `@param`.

Reply:
Done. Thanks John.